src/components/OutlinedCardList.tsx

```diff
@@ -27,7 +28,7 @@ const useStyles = makeStyles((theme: Theme) =>
     root: (props: CardProps) => ({
       marginTop: '10px',
       border: '1px solid',
-      borderColor: props.readOnly ? theme.palette.background.default : theme.palette.text.disabled,
+      borderColor: props.highlightedBorder ? theme.palette.primary.main : props.readOnly ? theme.palette.background.default : theme.palette.text.disabled,
```

Comment:
NIT: Can you put parentheses around the second ternary statement so it's easier to read? Otherwise nested ternaries can be pretty confusing to people who aren't used to reading JS

```
props.highlightedBorder ? theme.palette.primary.main : (props.readOnly ? theme.palette.background.default : theme.palette.text.disabled)
```